mava/systems/jax/launcher.py

```diff
@@ -27,6 +29,30 @@ class NodeType:
     courier = lp.CourierNode
 
 
+def copy_store(builder: SystemBuilder, multi_process: bool) -> SystemBuilder:
```

Comment:
Please rename this to something like `copy_store_in_builder`, to make its function more clear.

Reply:
Thanks @mmorris44 :slightly_smiling_face: It was a team effort :fire:  I changed the naming to `copy_builder`. As we are actually copying the entire builder except for store.program. Let me know if you agree.

Reply:
Ahh, my mistake. Yup, agreed then! I like the new name and file location.